### DISCUSSION TITLE
Getting "Notice: Image Download Token" in red text

### DISCUSSION THREAD

**Discussion by The0cdguy** [lxdware/lxd-dashboard#73] [Q&A]
I have a fairly new lxd dashboard installation that I'm using to administer 6 home lab servers - each running Debian 11.  Five of the servers are clustered and have separate cluster (10 mbit) and public (1 mbit) network interfaces.  When I use the GUI to download an image, the operation appears to be successful - the image is added to the list.  But, I get red warning text at the top of the browser window that reads: "Notice: Image Download Token".

When I go to Operations, I see that the task is still running.  By the way, when I download an image directly from a server console, none of this happens.  Am I using the image download feature incorrectly?  Is this a permissions/authentication/shared secret issue related to remotes and tokens?

**Comment by matthewalanpenning** [comment]
Yes, this happens on image downloads within a cluster, the download tokens are not cleared. I need to update the dashboard to correct this issue. I will put out a fix in the next update, If not this coming weekend, the following weekend. For now, you can delete the Image Download token operations that are left running,
